config/checkerframework/CallableStatement.astub

```diff
@@ -0,0 +1,91 @@
+package java.sql;
+
+import org.checkerframework.checker.nullness.qual.*;
+
+interface CallableStatement {
+  @Nullable String getString(int parameterIndex);
+  @Nullable BigDecimal getBigDecimal(int parameterIndex, int scale);
+  byte @Nullable [] getBytes(int parameterIndex);
```

Comment:
shouldn't this be `@Nullable byte[]`? The `byte[]` returned could be null, but items in the array cannot be null (they are primitives).

Reply:
See https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html#jls-9.7.4

Reply:
got it.
that seems is not obvious, but I guess once I see it enough it will not feel so foreign.